bluesky/run_engine.py

```diff
@@ -222,7 +226,7 @@ def __init__(self, md=None, *, loop=None, preprocessors=None,
 
         # Make a logger for this specific RE instance, using the instance's
         # Python id, to keep from mixing output from separate instances.
-        logger_name = "bluesky.RE.{id}".format(id=id(self))
+        logger_name = 'bluesky.RE.{id}'.format(id=id(self))
```

Comment:
As we did with `caproto.threading.client.Context`, we should remove the templated portion `.{id}` and use just one logger.

Reply:
That was my initial thinking. I was thinking you probably want to have id to do some fancy thing